### ISSUE TITLE
5e Gunslinger Archetype Issue

### ISSUE THREAD

**Issue by WarHermon** [Sklore/HL_DD_5e_Colab#679]
Hi,

Having an issue with guns on the Weapons tab not showing the attack modifier under the weapon's name. Also, even if you hover over the question mark section of the weapon, the modifier listed should be a +4, not a +2 (+2 for Dex modifier, +2 for Proficiency bonus). Tried different guns besides the Pepperbox and still have the same issue. I'm attaching some screenshots to give you some insight (using Print Screen, then, pasting to MS Paint; my apologies for this and dual monitors). Please let me know if you need any other information.

-Steve-

![gunslinger issue 1](https://user-images.githubusercontent.com/29740779/27612229-8615bbd8-5b63-11e7-8f99-a1a5a7001168.png)
![gunslinger issue 2](https://user-images.githubusercontent.com/29740779/27612228-861533e8-5b63-11e7-8a2d-9c3e0293e5fe.png)
![gunslinger other info 1](https://user-images.githubusercontent.com/29740779/27612224-860e64e6-5b63-11e7-89fc-762189073aee.png)
![gunslinger other info 2](https://user-images.githubusercontent.com/29740779/27612227-8613461e-5b63-11e7-8d85-e525740ef04b.png)
![gunslinger info 3](https://user-images.githubusercontent.com/29740779/27612226-86119f94-5b63-11e7-93c4-8f66d3dc1017.png)
![gunslinger other info 4](https://user-images.githubusercontent.com/29740779/27612225-861114e8-5b63-11e7-8aea-7a7010cf03f6.png)

**Comment by DarkHayZe** [comment]
Looks like you are using windows 7 or newer, in which case you have access to something called the snipping tool. It will allow you to take precise shots of your screen, and only what you want to show. If would be willing to recapture with that it would be greatly appreciated, my screen is only so big...lol

**Comment by DarkHayZe** [comment]
I am not having this issue, I did fix one thing on the pepperbox on my end, but the rest of them are loading fine. Do you have the most recent file COM_5ePack_Homebrew - Classes.user?

![image](https://user-images.githubusercontent.com/17392134/27612999-d3e7a5c4-5b5e-11e7-91cc-265391dd4812.png)

![image](https://user-images.githubusercontent.com/17392134/27613006-df180970-5b5e-11e7-8fe4-cb684b392864.png)

I will update the server side file with my update. But it should all be in working order.

**Comment by WarHermon** [comment]
Sure. However, I can't do a capture this way of what the question mark says if I hover or click on it, then use the snipping tool, as it disappears that way. So, I'm snipping a part of the 2nd pic I originally took to show what it says...

![capture 1](https://user-images.githubusercontent.com/29740779/27613069-8fac1a08-5b67-11e7-935a-c6b70a4b3ef5.PNG)
![capture 2](https://user-images.githubusercontent.com/29740779/27613073-8fada210-5b67-11e7-9d39-160811de037e.PNG)
![capture 3](https://user-images.githubusercontent.com/29740779/27613074-8faf4494-5b67-11e7-9a50-67d70d17a73f.PNG)
![capture 4](https://user-images.githubusercontent.com/29740779/27613068-8fa942b0-5b67-11e7-9e81-cda075d46a1b.PNG)
![capture 5](https://user-images.githubusercontent.com/29740779/27613070-8fac4348-5b67-11e7-98a6-487338e80827.PNG)
![capture 6](https://user-images.githubusercontent.com/29740779/27613071-8fac88a8-5b67-11e7-8740-9ed830985ded.PNG)

**Comment by WarHermon** [comment]
To answer your most recent question:

![capture 7](https://user-images.githubusercontent.com/29740779/27613126-da45ae26-5b67-11e7-88a8-f8ee5e8a8794.PNG)

**Comment by DarkHayZe** [comment]
I am not sure there has been an official push since the file was last updated. I do know that they are working on one now.

As to your capture problem, there should be a delay option to over come that. For future reference.

You can manually download the updated files by clicking the clone button on the code tab. and overwritting everything in your data directory.

**Comment by WarHermon** [comment]
Forgive me, but I have been searching for a while...the "Code Tab"?

**Comment by DarkHayZe** [comment]
![image](https://user-images.githubusercontent.com/17392134/27613556-d49d29d2-5b61-11e7-9f6d-81fc45e1439b.png)

**Comment by WarHermon** [comment]
Crud, was looking in Hero Lab itself.

I'll be honest, I'm not sure what to do from here. Clicking the "Clone or download" green button gives me different options. I'm not sure I should download the zip and put it in some directory on my computer, open in Desktop, etc. I even downloaded the GitHub Desktop, but haven't a clue what to do. I think I'm over my head on this.

**Comment by DarkHayZe** [comment]
Yes download the zip, and unpack it into your hero lab > data > 5e folder on your computer

**Comment by WarHermon** [comment]
Couldn't find the location at first. Had to go to Appearance And Personalization and choose "Show hidden files and folders" and get that taken care of. After that, went to my C: drive and found the "ProgramData" folder, which was hidden before. Then to Hero Lab>data>5e and took the contents that I unzipped and had them replace all of the old ones. Finally, I opened up Hero Lab itself and checked the character again: it's now working properly (in fact, it's a +6 mod due to the addition of the archery fighting style...this is legal - lol)

Thank you for your patience, DarkHayZe.  :)

-Steve-

![fixed](https://user-images.githubusercontent.com/29740779/27615983-3fe6e634-5b7a-11e7-8d70-a5218dd2be38.PNG)
